jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -366,7 +361,17 @@
       (setq obj-box-z-length (z-of-cube (send self :bbox->cube obj-box)))
       (setq obj-pos (v+ obj-pos (float-vector 0 0 (/ obj-box-z-length 2))))
       obj-pos))
-  (:try-to-pick-object-v4
+  (:try-to-pick-object
+    (arm obj-pos pinch-yaw &key (offset #f(0 0 0)) (grasp-style :suction))
```

Comment:
Sorry, my comment above is for  try-to-pinch-object.
I'll fix pinch-yaw of try-to-pick-object